pkg/apis/servicecatalog/validation/broker.go

```diff
@@ -98,13 +98,25 @@ func validateServiceBrokerSpec(spec *sc.ServiceBrokerSpec, fldPath *field.Path)
 				field.Required(fldPath.Child("authInfo"), "auth config is required"),
 			)
 		}
-
 	}
 
 	if spec.InsecureSkipTLSVerify && len(spec.CABundle) > 0 {
 		allErrs = append(allErrs, field.Invalid(fldPath.Child("caBundle"), spec.CABundle, "caBundle cannot be used when insecureSkipTLSVerify is true"))
 	}
 
+	if "" == spec.RelistBehavior {
+		allErrs = append(allErrs,
+			field.Required(fldPath.Child("relistDuration"),
+				"brokers must have a relistDuration"))
+	}
+
+	if spec.RelistBehavior == sc.ServiceBrokerRelistBehaviorDuration && spec.RelistDuration == nil {
+		allErrs = append(
+			allErrs,
+			field.Required(fldPath.Child("relistDuration"), "relistDuration must be set if behavior is set to Duration"),
+		)
+	}
+
```

Comment:
Do we want to enforce that `relistRequests` be non-negative.

Reply:
I had a similar though. Should the field be a `uint`? I that normally acceptable?

Reply:
`uint` is discouraged. All you need to do is add a validation check to ensure it is non-negative. And indicate that restriction in the comment.

Reply:
@staebler pushed an update to the comments indicating as such. Validation asserts greater than zero.